src/_zkapauthorizer/tests/test_replicate.py

```diff
@@ -186,3 +187,19 @@ def test_snapshot(self):
             conn_a,
             equals_database(conn_b),
         )
+
+
+class ReplicationServiceTests(TestCase):
+    """
+    Tests for ``_ReplicationService``.
+    """
+
+    def test_enable_replication_on_connection(self):
+        """
+        When the service starts it enables replication on its database connection.
+        """
+        conn = memory_connect("/foo/bar")
+        replicating_conn = with_replication(conn)
+        service = replication_service(replicating_conn)
+        service.startService()
+        self.assertThat(replicating_conn._replicating, Equals(True))
```

Comment:
It would probably be harder to scrap it right now than to scrap it eight weeks ago or 6 months from now.  It might be easier to scrap it right now than to *never* scrap it.  My preference is for 6 months from now. :)